### PR TITLE
Fixing multiple issues for start chat method and other fixes

### PR DESCRIPTION
**Pull request by sarojkpr**
1. Fixing `startChat` SDK method issues - unable to maximize a manimized chat using `startChat` SDK
2. Fixing `startChat` connecting to previous chat in popout mode
3. Fixing maximize events being raised on Start Chat button click for new chat
4. Fixing typo on messageXXXX middleware and warning
5. Fixing `lcw:startChat` not getting logged on `startChat` SDK call

### CODE REVIEW

File: CHANGE_LOG.md [hunk 1]
```diff
@@ -29,4 +29,7 @@ All notable changes to this project will be documented in this file.## [Unreleas
 - Updated the version of @microsoft/omnichannel-chat-components to @0.1.0-main.353ecff
 ### Fixed
 - Fixed hover customizations for Audio notification button
-
+### Fixed
```
Comment (charliewang95) [inline CHANGE_LOG.md:32]: Please put everything under "Unrelease". We haven't released any formal version

File: chat-widget/src/common/telemetry/TelemetryConstants.ts [hunk 2]
```diff
@@ -40,7 +40,9 @@ export enum BroadcastEvent {
     StartUnauthenticatedReconnectChat = "StartUnauthenticatedReconnectChat",
     EndChat = "EndChat",
     SetCustomContext = "SetCustomContext",
-    ChatRetrievedFromCache = "ChatRetrievedFromCache"
+    ChatRetrievedFromCache = "ChatRetrievedFromCache",
```
Comment (charliewang95) [inline chat-widget/src/common/telemetry/TelemetryConstants.ts:43]: Telemetry.ms updates please

File: chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx [hunk 3]
```diff
@@ -34,6 +33,10 @@ export const ChatButtonStateful = (props: IChatButtonStatefulParams) => {
         hideNotificationBubble: buttonProps?.controlProps?.hideNotificationBubble === true || state.appStates.isMinimized === false,
         unreadMessageCount: state.appStates.unreadMessageCount ? (state.appStates.unreadMessageCount > Constants.maximumUnreadMessageCount ? props.buttonProps?.controlProps?.largeUnreadMessageString : state.appStates.unreadMessageCount.toString()) : "0",
         onClick: async () => {
+            BroadcastService.postMessage({
```
Comment (charliewang95) [inline chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx:36]: Do we really need this?

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 4]
```diff
@@ -213,6 +240,11 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         if (state.appStates.conversationEndedByAgent) {
             endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter);
         }
+
+        //Listen to WidgetSize
+        BroadcastService.getMessageByEventName("WidgetSize").subscribe((msg: ICustomEvent) => {
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:245]: What is this used for?

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 5]
```diff
@@ -158,26 +160,51 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
...
+                persistedState.appStates.conversationState === ConversationState.InActive ||
+                persistedState.appStates.conversationState === ConversationState.Postchat) {
+                BroadcastService.postMessage({
+                    eventName: BroadcastEvent.ChatInitiated
+                });
                 prepareStartChat(props, chatSDK, state, dispatch, setAdapter);
+            } else {
+                dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: false });
+                BroadcastService.postMessage({
+                    eventName: BroadcastEvent.MaximizeChat,
```
Comment (charliewang95) [inline chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:184]: We don't need to differentiate maximizeChat scenarios between new chat and minimized chat, as discussed

Comment (sarojkpr) [reply chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:184]: This is used for calling Maximize chat on calling `startChat` on a minimized chat.

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx (hunk 4 above)
Comment (sarojkpr) [reply chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:254]: For restoring the widgetsize when maximizing chat.

File: chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx (hunk 3 above)
Comment (sarojkpr) [reply chat-widget/src/components/chatbuttonstateful/ChatButtonStateful.tsx]: removed this

File: chat-widget/src/common/telemetry/TelemetryConstants.ts (hunk 2 above)
Comment (sarojkpr) [reply chat-widget/src/common/telemetry/TelemetryConstants.ts:43]: Updated

**Review by charliewang95 — APPROVED** [review]

**Pull request merged**